src/objects/p5.Shader.js

```diff
@@ -0,0 +1,312 @@
+// Adapted by lightgl.js by evanw
+// https://github.com/evanw/lightgl.js/
+
+/**
+ * @module Shader 
+ * @submodule Shader 
+ * @requires core 
+ */
+define(function (require) {
+
+  var p5 = require('core');
+  require('p5.Graphics3D');
```

Comment:
If #720 lands before this does, this will need to be changed to `'3d/p5.Graphics3D'`. If this lands before #720 does, that PR will need to be rebased onto this one to account for the change.